src/main/webapp/style.css

```diff
@@ -273,7 +282,6 @@ tr:nth-child(even) {
 .sdkAnalysis {
   grid-column: 1;
   grid-row: row 3;
-  background-color: papayawhip;
```

Comment:
Would it be possible not to delete these just yet - at the moment I'm using them for testing as I add graphs (just so I can see what is working and what isn't as the graphs show up clearer against a coloured background) - thanks!

Reply:
Sure! No problem :D

Reply:
solved here https://github.com/googleinterns/step229-2020/pull/47/commits/794ab204e3639442e7688891bc2f6fd785bce6a4